resources/slack-openapi-patched.json

```diff
@@ -4237,6 +4237,7 @@
                         "description": "Authentication token. Requires scope: `admin.conversations:write`",
                         "in": "formData",
                         "name": "token",
+                        "required": true,
```

Comment:
Same comment as the other PR, this has nothing to do here IMO.

Reply:
fixed